lib/tasks/temp.rake

```diff
@@ -16,6 +16,44 @@ namespace :temp do
     end
   end
 
+  def backload_overdue(event_type, verbose)
```

Comment:
I get now that it's based on alerts being sent, rather than inspecting the list of events to see if the request is overdue directly (it caught me out because locally I hadn't sent the alerts, so I wasn't getting any items for overdue requests). Presumably this is intended to be more performant? If so, is it a big difference, because it seems clearer to call the method on the model rather than rely on the alerts having been sent?